joplin/api/schema.py

```diff
@@ -566,7 +607,7 @@ def resolve_page_type(self, info):
         return ServicePage.get_verbose_name().lower()
 
     def resolve_janis_url(self, info):
-        return self.janis_publish_url()
+        return self.janis_urls()
```

Comment:
this seems like it's resolving an array of urls as a graphene.String()

Reply:
@chiaberry this was the only reason I didn't hit approve

Reply:
oh thank you -- I took out the publish url, because it as returning the baseurl which janis doesnt need. Which come to think of it, I don't know that servicePageNode or any of the page nodes should need to resolve their urls

Reply:
naming is a little odd now because we're resolving janis_url as a list of urls instead of one url, but not a reason to stop a merge